src/lib/circuit_breaker/circuit_breaker_params.c

```diff
@@ -103,14 +103,8 @@ PARAM_DEFINE_INT32(CBRK_AIRSPD_CHK, 0);
 /**
  * Circuit breaker for flight termination
  *
- * Setting this parameter to 121212 will disable the flight termination action.
- * If configured, flight termination can be triggered as a failsafe action of the following failures:
- * - RC or data link is lost
- * - The vehicle moves outside the geofence
- * - The FailureDetector reports a failure (e.g.: attitude failure).
- * - GPS is lost, height estimate is not available and RC is lost
- * Flight termination is also triggered on the IO processor if FMU is lost.
- * => With this circuit breaker enabled, flight termination will never be exectued, even if configured.
+ * Setting this parameter to 121212 will disable the flight termination action if triggered
+ * by the FailureDetector logic or if FMU is lost.
```

Comment:
Oh gosh, why does it keep changing :see_no_evil:

Reply:
Maybe we can extend it, by being more explict and saying that flight termination will still be triggered when configured via RC loss action for example?